src/main/java/nl/tudelft/ti2206/group9/util/SaveGameParser.java

```diff
@@ -0,0 +1,77 @@
+package nl.tudelft.ti2206.group9.util;
+
+import nl.tudelft.ti2206.group9.level.State;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * @author Mathias
+ */
+public final class SaveGameParser {
+
+	/**
+	 * Private constructor.
+	 */
+	private SaveGameParser() { }
+
+	/** Playername. */
+	private static String playername;
+	/** Number of coins.*/
+	private static int coins;
+	/** Players highscore. */
+	private static double highScore;
+	/** Boolean to indicate whether the sound is enabled. */
+	private static boolean soundEnabled;
+
+	/**
+	 * Read a json savefile and store all data in the State class.
+	 * @param filePath path to the file to be parsed.
+	 */
+	public static void loadGame(final String filePath) {
+		try {
+			FileReader reader = new FileReader(filePath);
```

Comment:
*Auto-generated comment*

    Found reliance on default encoding in nl.tudelft.ti2206.group9.util.SaveGameParser.loadGame(String): new java.io.FileReader(String)